versioned_docs/version-V2/use-cases/private-voting.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 2
+---
+
+## Private voting overview
+
+Learn how Semaphore enables applications to do the following:
+
+- Register members as voters.
+- Allow members to vote anonymously.
+- Prove voter membership.
+- Record and prove votes.
```

Comment:
- Prove voter membership.
- Record and prove votes.

These two points could be included in the second one "- Allow members to vote anonymously."